### DISCUSSION TITLE
Option to show markdown editor in 'Preview' mode first

### DISCUSSION THREAD

**Discussion by PieturP** [directus/directus#20447] [Feature Requests]
### Summary

- I'd like the Markdown editors to be able to be rendered in 'Preview' mode upon editing an item with such an interface.

### Motivation

(Less technical) users will be able to see the rendered markdown content instead of the underlying markup code. This simplifies grasping existing content.

We do use Markdown extensively in our projects, but not all our users are really proficient with it.

### Detailed Design

Just like for example the options 'Preview font' and 'Edtior font', the Interface details for 'Markdown' fields should ideally have an option to toggle between 'Preview' (proposed) and 'Editor' (current) render / startup option.

### Requirements List

This feature would need a simple toggle or checkbox to switch the behavior of the rendering of the Markdown field.

### Drawbacks

It *could* make editing MD content slightly less obvious. 
(Although this could be mitigated by a simple click in the rendered content that could maybe toggle the mode as well as the already present mode button)

### Alternatives

n.a.

### Adoption Strategy

n.a.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
